src/Aquarius.Client/Samples/Client/create_service_models.sh

```diff
@@ -18,13 +18,15 @@ show_usage() {
 	exit_abort "$@"
 }
 
+GeneratorProjectPath=../../../SamplesServiceModelGenerator/SamplesServiceModelGenerator.csproj
 Generator=../../../SamplesServiceModelGenerator/bin/Release/net8.0/SamplesServiceModelGenerator.dll
 
 ServerName=$1
 OutputPath=$2
 
 command -v dotnet >/dev/null 2>&1 || exit_abort "This script requires the .NET CORE runtime. Grab it from here: https://www.microsoft.com/net/download"
```

Comment:
```suggestion
command -v dotnet >/dev/null 2>&1 || exit_abort "This script requires the .NET 8 runtime. Grab it from here: https://www.microsoft.com/net/download"
```

this bit of code looks useful for the java builder, especially since the default assumption (to me) would be the java sdk is built with java (instead of .net)